dali/operators/reader/fits_reader_op.cc

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2020-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <string>
+
+#include "dali/core/backend_tags.h"
+#include "dali/core/static_switch.h"
+#include "dali/kernels/slice/slice_cpu.h"
+#include "dali/kernels/slice/slice_flip_normalize_permute_pad_cpu.h"
+#include "dali/kernels/transpose/transpose.h"
+#include "dali/operators/reader/fits_reader_op.h"
+#include "dali/pipeline/data/backend.h"
+
+namespace dali {
+namespace detail {
+inline int FitsReaderOutputFn(const OpSpec &spec) {
+  return spec.HasArgument("hdu_indices") ?
+             spec.GetRepeatedArgument<std::string>("hdu_indices").size() : 1;
+}
+
+}  // namespace detail
+
+DALI_REGISTER_OPERATOR(experimental_readers__Fits, FitsReaderCPU, CPU);
+
+DALI_SCHEMA(experimental_readers__Fits)
```

Comment:
```suggestion
DALI_REGISTER_OPERATOR(experimental__readers__Fits, FitsReaderCPU, CPU);

DALI_SCHEMA(experimental__readers__Fits)
```
`__` - double underscore is needed.

Reply:
On it!